install_config/install/advanced_install.adoc

```diff
@@ -315,6 +315,30 @@ node1.example.com openshift_node_labels="{'region': 'primary', 'zone': 'east'}"
 ----
 ====
 
+Nodes only receive labels if you explicitly add them in the inventory using
+`*openshift_node_labels*` as described below. `*region=infra*` is the default
+selector. If you create nodes with that selector, then the router and registry
+will be installed.
+
+====
+----
+# default selectors for router and registry services
+# openshift_router_selector='region=infra'
+# openshift_registry_selector='region=infra'
+----
+====
```

Comment:
Those lines are likely taken directly from the configuration file.  It's common for configuration files to show the defaults by comments.  That way should an admin wish to change them they simply uncomment the line and then change the value.  If they don't change the value then there really isn't any point to uncomment them.  Personally, I would leave these lines as is.